src/main/java/com/example/Ingress_lab/service/concrete/LessonServiceHandler.java

```diff
@@ -0,0 +1,61 @@
+package com.example.Ingress_lab.service.concrete;
+
+import com.example.Ingress_lab.dao.repository.LessonRepository;
+import com.example.Ingress_lab.mapper.Mapper;
+import com.example.Ingress_lab.model.enums.LessonStatus;
+import com.example.Ingress_lab.model.request.LessonRequest;
+import com.example.Ingress_lab.model.response.LessonResponse;
+import com.example.Ingress_lab.service.abstraction.LessonService;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+@Service
+@RequiredArgsConstructor
+public class LessonServiceHandler implements LessonService {
+    private final Mapper mapper;
+    private final LessonRepository lessonRepository;
+
+    @Override
+    public void createLesson(LessonRequest request) {
+        lessonRepository.save(mapper.toLessonEntity(request));
+    }
+
+    @Override
+    public LessonResponse getLessonById(Long id) {
+        return lessonRepository
```

Comment:
1. create private fetchIfExist method
```
private LessonEntity fetchIfExist(Long id) {
  return lessonRepository.findById(id)
                                        .orElseThrow(RuntimeException::new);
}
```
2. use this method in deleteById, updateById and findByid methods